packages/grid/src/Grid.tsx

```diff
@@ -453,7 +453,6 @@ export const Grid = function Grid<T>(props: GridProps<T>) {
     rowIdx: number,
     colIdx: number
   ) => {
-    setTimeout(() => {
```

Comment:
What is the reason you are removing this setTimeout? It was originally put here for a reason, to counter a race condition. I forgot the exact scenario but will need to do some manual QA locally to see if anything is broken.

Reply:
Ah, turns out the test caught the scenario that has a race condition.

Reply:
More context: removing `setTimeOut` results in one of the tests for keyboard navigation failing, we believe `setTimeOut` was initially added to counter virtualization lag, i.e. when we focus on a cell that hasn't been rendered yet

Reply:
@ZarkoPernar @feliciackh  Got it . The Reason to remove this was as it pushes it in event queue so it dosent run in sync and due to that onBlur function is not gettin called as it focus after a while .